lib/winston-graylog2.js

```diff
@@ -53,6 +53,9 @@ function prepareMeta(meta) {
     });
   }
 
+  // merge the meta data with the statically configured metadata
```

Comment:
This comment is kind of redundant, it kinds of repeat exactly what the code says.